### PR TITLE
Created index.html and style.css

### PR DESCRIPTION
**Pull request by nicupop729**
## Hello Microverse

In this branch I have done the following:
- set up a linter for HTML and CSS
- created index.html file
- created style.css file
- created but ignored test.md file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (karvel-code) [inline .gitignore]: - [x] Please add `node_modules` in your `.gitignore` file since it is among the project requirements

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @nicupop729 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 ## To Highlight
- Your linter checks are passing. 
- You have followed git-flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (nicupop729) [reply .gitignore]: Thank you very much for the feedback!

**Comment by nicupop729** [comment]
Hi @karvel-code
I have added to .gitignore node_modules.
Thank you very much for the feedback!

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @nicupop729,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**